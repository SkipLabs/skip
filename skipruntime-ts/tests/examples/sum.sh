#!/bin/bash
set -e
cd ../../examples

if [ "$#" -ne 2 ]; then
    echo "Usage: $0 <stdout target> <stderr target>"
fi

node dist/sum.js >/dev/null &
sleep 0.25 # give a moment for service to spin up
node dist/sum-client.js >"$1" 2>"$2"

jobs -p | xargs kill
