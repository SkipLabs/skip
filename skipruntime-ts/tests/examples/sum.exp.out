Init []
Update [ [ 'v1', [ 2 ] ] ]
Update [ [ 'v1', [ 5 ] ] ]
Update [ [ 'v1', [ 3 ] ] ]
Update [ [ 'v1', [ 5 ] ], [ 'v2', [ 6 ] ] ]
Update [ [ 'v1', [ 11 ] ] ]
